analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -791,7 +549,7 @@ private static void MethodName(object[] table, object[][] aoTable2, ref Dictiona
 
             if (list.Count > 0)
             {
-                list.Sort(); // Compliant. This used to be a FP
+                list.Sort();    // Compliant - used to be an FP
```

Comment:
For simplicity, we don't need to drag history of the old SE with us
```suggestion
                list.Sort();    // Compliant
```